paddle/platform/profiler.cc

```diff
@@ -288,9 +325,10 @@ void ParseEvents(std::vector<std::vector<Event>>& events,
         }
       }
     }
-    // average time
+    // average time and memory used
     for (auto& item : event_items) {
       item.ave_time = item.total_time / item.calls;
+      item.ave_memory_used = item.total_memory_used / item.calls;
```

Comment:
I don't think `item.ave_memory_used` is necessary. 
`average memory used` is confusing.

Reply:
I tend to count the memory used in the most case.

Reply:
deleted.